### PR TITLE
feat: support raw conversion in envs without `Buffer`

### PR DESCRIPTION
**Pull request by itpropro**
### 🔗 Linked issue

#142

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme, or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [X] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Replaced Buffer with native implementation to also run on workers and other environments. It's also faster than Buffer, but in the context of unstorage, the improvement is not significant.

### 📝 Checklist

- [X] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by itpropro** [comment]
The netlify tests seem to be very flaky and slow, we should probably disable them (95% of them ran into timeout on my machine) until the driver is fixed or a faster way of testing is found.

**Comment by GalacticHypernova** [comment]
How would this translate to getItemRaw with the web streams? Would it still be possible for the response of the image to be rendered correctly?

File: src/_utils.ts [hunk 1]
```diff
@@ -72,6 +65,34 @@ export function deserializeRaw(value: any) {
     // Return unknown strings as-is
     return value;
   }
-  checkBufferSupport();
-  return Buffer.from(value.slice(BASE64_PREFIX.length), "base64");
+  return genBytesFromBase64(value.slice(BASE64_PREFIX.length));
+}
+
+// Base64 utilities - Waiting for https://github.com/unjs/knitwork/pull/83 // TODO: Replace with knitwork imports as soon as PR is merged
```
Comment (manniL) [inline src/_utils.ts:71]: Can now be used via https://github.com/unjs/undio/pull/8

Comment (pi0) [reply src/_utils.ts:71]: Yes indeed. The feature is not releaed yet and I ned to complete node {Buffer} support but we shall adopt undio.

File: src/drivers/http.ts [hunk 2]
```diff
@@ -33,14 +33,14 @@ export default defineDriver((opts: HTTPOptions) => {
...
-      const value = await _fetch(r(key), {
+      const value = await _fetch.native(r(key), {
         headers: {
           accept: "application/octet-stream",
           ...opts.headers,
           ...topts.headers,
         },
       });
-      return value;
+      return new Uint8Array(await value.arrayBuffer());
```
Comment (pi0) [inline src/drivers/http.ts:43]: This is nice change but has to be split

File: src/_utils.ts (hunk 1 above)
Comment (pi0) [reply src/_utils.ts:71]: will add a TODO to refactor once undio stable

**Review by pi0 — APPROVED** [review]
Thanks ❤️

File: src/drivers/http.ts (hunk 2 above)
Comment (pi0) [reply src/drivers/http.ts]: ~> #527

**Pull request merged**